holidayextras/partner-api-docs
# IPP-336 initial payment gateway documentation

This PR adds documentation for the new Payment Gateway Service. This is an externally accessible payment api, that can handle payments and refunds.

This details the 2 endpoints available in the Payment Gateway, and also gives a step by step guide on how to get setup to make these requests.

To Test:

Read through the documentation, following instructions (please comment if anything isnt obvious so we can update the documentation).

Attempt to make a successful payment - see myself or Tom P for credentials
Attempt to make a successful refund using the paymentId created in the above step
